pages/landing/enterprise-eks/pricing/index.html

```diff
@@ -31,10 +31,10 @@
       - name: Basic networking
         description: Support for AWS VPC based networking for Kubernetes Pods.
         plans: ["Standard", "Enterprise"]
-      - name: Cluster auto scaling
+      - name: Basic auto scaling
```

Comment:
Similar to above, I switched this based on https://github.com/gruntwork-io/gruntwork-io.github.io/pull/579#discussion_r686719602 but again I felt a bit weird calling this "Basic" because it's fairly advanced (most basic auto scaling is CPU based, but this is based on Pod scheduling). It also doesn't include pod auto scaling at all, and I was worried calling this "Basic auto scaling" implies that - I have a hunch most people will expect pod auto scaling before cluster auto scaling.

Reply:
Well, the implication here is "basic" as in "basic subscription" and not as in "simple functionality." But I suppose there is a little ambiguity there.

Perhaps we can make that explicit? What if we replace the word "basic" with "standard" and the word "advanced" with "enterprise": e.g., "Standard Monitoring" and "Enterprise Monitoring" or "Standard Auto Scaling" and "Enterprise Auto Scaling"?  So now it 100% reflects the subscription types, and hopefully removes ambiguity with how complicated the feature is?

Reply:
I like that, so let's go with it.

Reply:
Done: 7df5161